NEWS.md

```diff
@@ -54,7 +54,7 @@ tidyr contains four new functions to support **rectangling**, turning a deeply n
 
 `unnest_longer()` and `unnest_wider()` make it easier to unnest list-columns of vectors into either rows or columns (#418). `unnest_auto()` automatically picks between `_longer()` and `_wider()` using a little magic.
```

Comment:
Do we have a PR number of "using a little magic"? Or is that why we're using the phrase?

Reply:
There's no PR because it's all new code. Maybe "heuristics based on the presence of common names" would be better?